app/javascript/packs/articlePage.jsx

```diff
@@ -122,43 +117,3 @@ getCsrfToken().then(async () => {
       '<p className="color-accent-danger">Unable to load Comment Subscription component.<br />Try refreshing the page.</p>';
   }
 });
-
-// Initialize the profile preview functionality
-const profilePreviewTrigger = document.getElementById(
-  'profile-preview-trigger',
-);
-
-const dropdownContent = document.getElementById('profile-preview-content');
-
-if (profilePreviewTrigger?.dataset.initialized !== 'true') {
-  initializeDropdown({
-    triggerElementId: 'profile-preview-trigger',
-    dropdownContentId: 'profile-preview-content',
-    onOpen: () => {
-      dropdownContent?.classList.add('showing');
-    },
-    onClose: () => {
-      dropdownContent?.classList.remove('showing');
-    },
-  });
-
-  profilePreviewTrigger.dataset.initialized = 'true';
-}
-
-const targetNode = document.querySelector('#comments');
-targetNode && embedGists(targetNode);
```

Comment:
Need to add these lines back in as they're not for the preview card (should fix one of the failing specs 🙂 )

Reply:
wooohooo, all checks green! thanks soooo much for help @aitchiss !